src/pages/index.js

```diff
@@ -349,10 +339,10 @@ const HomeCard = ({ cardData, index }) => {
             initial="hide"
             animate={inView ? 'show' : 'hide'}
           >
-            <motion.h5
+            <motion.h2
               variants={basicStaggerChild}
               className="knockout-bold text-uppercase"
-            >{`${index + 1}. ${cardData.cardTitle}`}</motion.h5>
+            >{`${index + 1}. ${cardData.cardTitle}`}</motion.h2>
             <motion.div variants={basicStaggerChild}>
               <MDXRenderer>{cardData.cardHeading}</MDXRenderer>
```

Comment:
these are set as heading level 4 in MDX frontmatter.  can we drop the heading 4 markdown in the frontmatter (`####`) and wrap this `MDXRenderer` in an `<h4>`?